src/components/App.tsx

```diff
@@ -18,8 +33,7 @@ class App extends React.Component<RouteProps, State> {
     super(props);
     this.logName = this.getLogNameFromRoute(props);
     this.state = {
-      headerText: `Loading board ${this.logName}...`,
-      logEntries: []
+      isLoading: true
```

Comment:
Don't you need to specify the type of the `state` as `readonly State`?

Reply:
It's already specified on the base class - `React.Component`. You only need to specify it when overriding.